app/code/Magento/Eav/Model/AttributeDataFactory.php

```diff
@@ -8,21 +8,19 @@
 
 /**
  * EAV Entity Attribute Data Factory
- *
- * @author      Magento Core Team <core@magentocommerce.com>
  */
 class AttributeDataFactory
 {
-    const OUTPUT_FORMAT_JSON = 'json';
-    const OUTPUT_FORMAT_TEXT = 'text';
-    const OUTPUT_FORMAT_HTML = 'html';
-    const OUTPUT_FORMAT_PDF = 'pdf';
-    const OUTPUT_FORMAT_ONELINE = 'oneline';
-    const OUTPUT_FORMAT_ARRAY = 'array';
+    public const OUTPUT_FORMAT_JSON = 'json';
+    public const OUTPUT_FORMAT_TEXT = 'text';
+    public const OUTPUT_FORMAT_HTML = 'html';
+    public const OUTPUT_FORMAT_PDF = 'pdf';
+    public const OUTPUT_FORMAT_ONELINE = 'oneline';
+    public const OUTPUT_FORMAT_ARRAY = 'array';
 
     // available only for multiply attributes
```

Comment:
Maybe remove this line and add it to line 23

Reply:
Thanks for the suggestion. I've applied this change in https://github.com/magento/magento2/pull/37002/commits/56da55e536e295ee8d51477e4d0e1c00ef108d3a